meson.build

```diff
@@ -1,6 +1,9 @@
 project(
     'navigation', 'cpp', 'c',
-    default_options: [ 'cpp_std=c++17' ]
+    default_options: [
+        'cpp_std=c++17',
+        'werror=true'
```

Comment:
I don't support this. Yes this forces our devs to awknowledge errors, but no I don't want to have to define an error suppression for those useless warnings, or warnings during dev.